### ISSUE TITLE
Naming

### ISSUE THREAD

**Issue by znicholls** [njleach/FaIR_v2-0#1]
tl;dr

It can't be called FaIR v2.0. We need a new name, my suggestion is 'Minimal Equation Set Simple Climate Model' as `messcm` is available on pypi.

---

In order for @chrisroadmap to be able to sensibly version FaIR (using [semantic versioning](https://www.geeksforgeeks.org/introduction-semantic-versioning/)) and to not drive all users insane, we can't use the name fair (every set of numbers X.Y.Z needs to be available for FaIR to use). I know this is annoying, but there are now multiple papers which use FaIR to point to @chrisroadmap's version and so I think that ship has sailed.

I'd suggest something which makes clear what this is trying to pull off, hence the above. @njleach and @stujen probably worth sorting this out before moving any further as it will require altering the repository name which is something you want to do sooner rather than later.

**Comment by rgieseke** [comment]
Agreeing with @znicholls - this would get very confusing!

In the repo you already have UnFaIRv2.0, in other repos 5FaIR, so maybe something along these lines could show the relationship but avoid consusion! FaIR5, FivaIR ...?
